src/test/java/io/debezium/connector/yugabytedb/TestHelper.java

```diff
@@ -426,7 +436,12 @@ public static String getNewDbStreamId(String namespaceName, String tableName) th
             throw new NullPointerException("No table found with the name " + tableName);
         }
 
-        return syncClient.createCDCStream(placeholderTable, namespaceName, "PROTO", "IMPLICIT").getStreamId();
+        String dbStreamId = syncClient.createCDCStream(placeholderTable, namespaceName,
```

Comment:
Can this statement throw an exception? I suggest wrapping it `try ... finally`

Reply:
Yes, this statement can throw an exception. However, the function itself checks this exception:
```
public static String getNewDbStreamId(String namespaceName, String tableName) throws Exception {
...
}
```

Is a `try...finally` block still needed?

Reply:
Then the client won't be closed.